app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  # validates :name, :username, :email
```

Comment:
Remove comments, please.

Reply:
Hi! Thanks for pointing it out! We have fixed the issue by adding a has_many relationship to the post model. We added comments to Posts 1 and 2 (post_id = 1, post_id = 2). Post 2 has 2 comments so you can call comments.first or comments.last to retrieve them. 
Happy coding!